### DISCUSSION TITLE
打开软件设置，提示出错浏览器 STATUS_INVALID_IMAGE_HASH

### DISCUSSION THREAD

**Discussion by Bruceluufly** [shuax/MouseInc.Settings#172] [General]
浏览器版本：版本 93.0.4577.63 耍下定制
软件版本：2.11.7
![image](https://user-images.githubusercontent.com/90894960/136128674-89ab02ff-222b-48a6-bba0-19c4851e4386.png)
360已经关闭

**Comment by Fallyn292** [comment]
我也是，不知道咋解决T T

**Comment by shuax** [comment]
那就装个webview2用离线包吧

**Comment by shuax** [comment]
https://github.com/shuax/MouseInc.Settings/releases
